wrappers/s2i/python/Makefile

```diff
@@ -3,8 +3,8 @@ SHELL:=/bin/bash
 
 PYTHON_VERSION=3.8.10
 
-CONDA_DOWNLOAD_VERSION=4.7.12
-CONDA_VERSION=4.10.3
+CONDA_DOWNLOAD_VERSION=py38_4.12.0
```

Comment:
For context, this is only about the python in downloaded conda, we still install desired version during the final image build.

Reply:
Would it be worth adding a comment with this?